src/Sylius/Bundle/ShippingBundle/Form/Type/ShippingMethodType.php

```diff
@@ -73,6 +74,10 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'type' => 'sylius_shipping_method_translation',
                 'label' => 'sylius.form.shipping_method.translations',
             ])
+            ->add('position', IntegerType::class, [
```

Comment:
It's the recommended way for Symfony 2.8+, we shouldn't use deprecated features if we can (we can for `integer` but not for `sylius_shipping_category_choice`).